Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1178,4 +1179,59 @@
     <string name="test_api_body_other_keys">"Other key"</string>
     <!-- NOTR -->
     <string name="test_api_calculate_risk_level">"Calculate Risk Level"</string>
+
+    <string name="country_name_all">Alle Länder</string>
```

Comment:
Add Translation comments to each new string, e.g.
```
<!-- XTXT: interoperability: countrylist - the user sees a list of all available countries -->
```

Reply:
Comments added to each string